test/unit/lib/stringUtils.js

```diff
@@ -39,4 +39,34 @@ describe('stringUtils', () => {
       expect(word).to.equal('LS1');
     });
   });
+
+  describe('joinWithCommas', () => {
+    it('should return no commas or and for a single word', () => {
+      const words = ['One'];
+      const joined = stringUtils.joinWithCommas(words);
+
+      expect(joined).to.equal('One');
+    });
+
+    it('should return an and for a two words', () => {
```

Comment:
For consistency with the last two tests, can the and be surrounded by `'`?
No need for the `a`.